content/using/install.md

```diff
@@ -8,24 +8,19 @@ aliases = ["/docs/getting-started/"]
 
 Urbit is not yet ready for everyday users — but if you're technically inclined or generally intrepid, feel free to try it out. It's a good place to explore.
 
-In order to get going on the network, you'll need an [Urbit ID](#id). To try the live network out with a disposable identity, you can always [create a comet](@/using/operations/creating-a-comet.md).
+There are two types of "ships" (Urbit instances) that typical person to use Urbit with: a **comet**, or a **planet**. Comets are disposable identities that anyone can make for free, making them simpler boot with, so we'll start with instructions on using one of those. Then, we'll continue onto instructions on using a planet.
 
-## Getting an Urbit ID {#id}
+If you know that you want to use a planet, a more permanent identity that requires an Urbit ID, follow the [installation instructions below](#urbit), and then skip to the section on [getting an Urbit ID](#id).
 
-For an introduction to Urbit IDs and the address space more generally, see [Urbit ID](@/understanding-urbit/urbit-id.md) in [Understanding Urbit](@/understanding-urbit/_index.md).
+## Table of Contents
 
-While Tlon does not currently sell or distribute Urbit IDs, there are still a few ways to get your own Urbit ID:
+- [Install Urbit](#Urbit)
+- [Booting a comet](#comet)
+- [Using an Urbit ID](#id)
+- [Using the web interface](#landscape)
```

Comment:
Table of contents is already provided in the sidebar tho?

Reply:
Right, gonna make a few more changes